elasticdl/python/master/optimizer_wrapper.py

```diff
@@ -116,10 +118,17 @@ def __init__(self, opt, kv_store_endpoint, embedding_dims):
                 {layer name: `embedding_dim`} where layer name is the
                 name of ElasticDL embedding layer and `embedding_dim`
                 is the output dimension of corresponding embedding layer.
+            need_create_var: A python bool. If `need_create_var` is True,
+                OptimizerWrapper creates temporary variables for embedding
```

Comment:
What does temporary variables mean here? Also maybe add comment on the relationship with `use_async`

Reply:
1. temporary variables means that `OptimizerWrapper` creates these variables every iteration. In one iteration, `OptimizerWrapper` creates these variables before calling optimizer's `apply_gradients` function and deletes these variables after calling `apply_gradients`.
2. Goot point. Putting `need_create_var` in the constructor will confuse users. Users only care about whether `OptimizerWrapper` is used in synchronous or asynchronous situation. I will put `use_async` argument here.